README.md

```diff
@@ -0,0 +1,29 @@
+# Ember-proxy-controllers
+
+Ember.ObjectController and Ember.ArrayController extracted as an standalone addon.
+
+## Installation
+
+Run `ember install ember-proxy-controllers` and you're good to go
+
+## Usage
+
+Import the classes from the addon and extend them as with the old `Ember.ArrayController`
+and `Ember.ObjectController`
+
+```js
+// app/controllers/posts.js
+import { ArrayController } from 'ember-proxy-controllers'
```

Comment:
thoughts on instead having:

* `ember-controllers/array`
* `ember-controllers/object`

as at some point we will likely add the `basic` controller to this list-aswell